libs/features/georoutingv2/georoutingv2.js

```diff
@@ -293,10 +308,23 @@ export default async function loadGeoRouting(conf, createTagFunc, getMetadataFun
   if (storedLocale || storedLocale === '') {
     // Show modal when url and cookie disagree
     if (urlLocale.split('_')[0] !== storedLocale.split('_')[0]) {
-      const localeMatches = json.georouting.data.filter((d) => d.prefix === storedLocale);
-      const details = await getDetails(urlGeoData, localeMatches, json.geos.data);
+      const localeMatches = json.georouting.data.filter(
+        (d) => d.prefix === storedLocale,
+      );
+      const details = await getDetails(
+        urlGeoData,
+        localeMatches,
+        json.geos.data,
+      );
```

Comment:
Is this formatting being done by a code editor extension or personal preference? I'd argue it creates a bit of noise in the PR and adds a few more lines without a real benefit. Same comment for L275 & L291.

Reply:
This was done by IDE extension for formatting. I have reverted it. However didn't reverted the one on line 275 as that was done to fix below linting issue
<img width="1516" alt="Screen Shot 2023-07-28 at 6 36 09 AM" src="https://github.com/adobecom/milo/assets/69535463/207a9420-7f42-4857-b6a2-4fb0a4dce15e">